lessons/specifics/mnesia.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: page
+title: Mnesia 
+category: specifics
+order: 4
+lang: en
+---
+
+Mnesia is a heavy duty real-time distributed database management system.
+
+
+## Table of Contents
+
+- [Overview](#overview)
+- [When to use](#when-to-use)
+- [Schema](#schema)
+- [Nodes](#nodes)
+- [Starting Mnesia](#starting-mnesia)
+- [Creating Tables](#creating-tables)
+- [The Dirty Way](#the-dirty-way)
+- [The Traditional Way](#the-traditional-way)
+
+
+## Overview
+
+Mnesia is a Database Management System (DBMS) that ships with the Erlang Runtime System which we can use naturally with Elixir. The Mnesia *relational and object hybrid data model* is what makes it suitable for developing distributed applications of any scale.
+
+## When to use
+
+When to use a particular piece of technology is often a confusing pursuit. If you can answer 'yes' to any of the following questions, then this is a good indication to use Mnesia over ETS or DETS.
+
+  - Do I need to roll back tranactions?
+  - Do I need an easy to use syntax for reading and writing data?
+  - Should I store data accross multiple nodes, rather than one?
+  - Do I need a choice wheere to store information (RAM or disk)?
+
+## Schema
+
+As Mnesia is part of the the Erlang core, rather than Elixir, we have to access it with the colon syntax (See Lesson: [Erlang Interoperability](https://elixirschool.com/lessons/advanced/erlang/)) as so:
+
+```shell
+
+iex> :mnesia.create_schema([node()])
+
+# or if you prefer the Elixir feel...
+
+iex> alias :mnesia, as: Mnesia
```

Comment:
Yeah, sure we can add some stuff on Amnesia, however it hasn't (more or less) been officially adopted by the community or have any official docs. This was my reasoning for not including it into the lesson.

I know we already discussed including 3rd party library lessons such as Poolboy. Maybe it should go in this category rather than coupled with the Mnesia lesson. 

We can discuss this further on slack or skype :smile: